core/src/apps/cardano/seed.py

```diff
@@ -65,32 +62,20 @@ async def get_keychain(ctx: wire.Context, namespace) -> Keychain:
     return keychain
 
 
-def _get_root_dict_key(namespace):
+def _get_root_cache_key(namespace):
     if namespace == BYRON_SEED_NAMESPACE:
-        return BYRON_ROOT_DICT_KEY
+        return cache.APP_CARDANO_BYRON_ROOT
     if namespace == SHELLEY_SEED_NAMESPACE:
```

Comment:
I would use `elif` and `else` here, it is safer (w.r.t. future code modifications) and makes the intent clearer.